docs/user_guide/hopsworks/newProject.rst

```diff
@@ -46,3 +48,20 @@ project and share your datasets or remove existing ones. Using the
 ``Metadata Designer`` you can attach more intuitive metadata to your
 project. Also, in the project's menu you can always see the current
 cluster utilization.
+
+
+Project name reserved words
+============================
+
+PROJECTS, HOPS-SYSTEM, HOPSWORKS, INFORMATION_SCHEMA, AIRFLOW, GLASSFISH_TIMERS, GRAFANA, HOPS, METASTORE, 
```

Comment:
I believe these lists will quickly become outdated, how are we going to keep them updated? I think if the users get a clear error like "cannot use name XYZ" then that's adequate.

cc @SirOibaf

Reply:
If it needs updating it will be the same in the code. We keep all these in settings. @SirOibaf suggested we link to the docs.

Reply:
Maybe we can add a comment in the Settings.java to remind people that if they change that list, they should also change the documentation.

Reply:
sounds good